packages/python/plotly/plotly/express/data/__init__.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import absolute_import
+from plotly.data import *
+
```

Comment:
should we add a short docstring here like in the top-level package?

```
"""
Built-in datasets for demonstration, educational and test purposes.
"""
```

Reply:
done